### PR TITLE
Add ServerRequest::getPath().

### PR DESCRIPTION
**Pull request by ADmad**
`ServerRequest::getRequestTarget()` is not a valid alternative for deprecated
`ServerRequest::$url` as value returned by the former includes "/" prefix and
request query string.

### CODE REVIEW

File: src/Http/ServerRequest.php [hunk 1]
```diff
@@ -2344,6 +2344,23 @@ public function getRequestTarget()
         return $target;
     }
 
+    /**
+     * Get the path of current request.
```
Comment (markstory) [inline src/Http/ServerRequest.php]: Should we mention that this is missing the leading and trailing slashes?

**Review by markstory — COMMENTED** [review]
Looks good. Tests are failing in PHP 5.6

File: src/Http/ServerRequest.php (hunk 1 above)
Comment (inoas) [reply src/Http/ServerRequest.php]: Shouldn't it include the leading slash at least?

Comment (ADmad) [reply src/Http/ServerRequest.php]: @inoas This is meant to be a replacement for `Request::$url` which does not include leading slash either.

That said I personally have almost always used that property after adding leading slash, so many in 4.0 we can modify the method to include leading slash.

**Comment by ADmad** [comment]
@markstory Docblock updated. Not sure why the test is failing on 5.6. The issue is probably in the code which generates `$request->uri` instance, not in the method I added. `$request->uri()->getPath()` is supposed to return `/` if you path has been specifically set.

File: src/Http/ServerRequest.php (hunk 1 above)
Comment (markstory) [reply src/Http/ServerRequest.php]: We could have the leading slash now. Adding it later is a subtle change that is likely going to cause problems. Doing it now will make the minor difference part of the upgrade process.

Comment (ADmad) [reply src/Http/ServerRequest.php]: @markstory I agree. Done so.

**Pull request merged**